### ISSUE TITLE
Fiberflat header documentation

### ISSUE THREAD

**Issue by julienguy** [desihub/desidatamodel#191]
The fiberflat header has a keyword SKYLEVEL that is no present in the data.
We will need to update this information.

**Comment by weaverba137** [comment]
@julienguy, there are several possible 'fiberflat' files. Could you please indicate the specific data model file that has this issue?

Also, when you say this keyword is not present, do you mean it has never been present now and through the entire history of DESI? 

The reason I'm asking is that the data model file may have been generated using a fiberflat file that did have that keyword. In that case we should mark the keyword as optional, rather than removing it entirely.

**Comment by weaverba137** [comment]
@julienguy, there's a question on here from back in April. Could you please take a look at this?

**Comment by julienguy** [comment]
Hi, there is no SKYLEVEL entry in the headers of the data files described by:
```
./doc/DESI_SPECTRO_REDUX/SPECPROD/calibnight/NIGHT/fiberflatnight-CAMERA-NIGHT.rst
./doc/DESI_SPECTRO_REDUX/SPECPROD/exposures/NIGHT/EXPID/fiberflatexp-CAMERA-EXPID.rst
./doc/DESI_SPECTRO_REDUX/SPECPROD/exposures/NIGHT/EXPID/stdstars-SPECTROGRAPH-EXPID.rst
./doc/DESI_SPECTRO_REDUX/SPECPROD/exposures/NIGHT/EXPID/fiberflat-CAMERA-EXPID.rst
```
I guess the ICS is not writing this keyword for calibration observations. I wanted to point that out in case we wanted to refine the data model description.

**Comment by weaverba137** [comment]
@julienguy, thank you. That sounds like `SKYLEVEL` appears in *some* files but not *all* files, in which case we can mark it as optional. Do you agree?

**Comment by weaverba137** [comment]
Actually, `SKYLEVEL` is already marked as optional in all of those files, but I also noticed that the description had a missing `[` so this will be fixed in PR #199.

**Comment by weaverba137** [comment]
Closed by #199.
